mgmtsystem/hawkular.py

```diff
@@ -0,0 +1,89 @@
+from collections import namedtuple
+from rest_client import ContainerClient
+
+"""
+Related yaml structures:
+
+[cfme_data]
+management_systems:
+    hawkular:
+        name: My hawkular
+        type: hawkular
+        hostname: 10.12.13.14
+        port: 8080
+        credentials: hawkular
+        authenticate: true
+        rest_protocol: http
+
+[credentials]
+hawkular:
+    username: admin
+    password: secret
+"""
+
+Feed = namedtuple('Feed', ['id', 'name', 'path'])
+ResourceType = namedtuple('ResourceType', ['id', 'name', 'path'])
+Server = namedtuple('Server', ['id', 'name', 'path'])
+
+
+class Hawkular(object):
+    """Hawkular management system
+
+    Hawkular REST API method calls.
+    Will be used by cfme_tests project to verify Hawkular content shown in CFME UI
+
+    Uses novaclient.
```

Comment:
copied from other class, so it's not some general implementation in ContainerClient, removing